src/waveform/renderers/waveformsignalcolors.cpp

```diff
@@ -134,6 +140,30 @@ void WaveformSignalColors::fallBackDefaultColor() {
     fallBackFromSignalColor();
 }
 
+QColor WaveformSignalColors::rgbaColorFromString(QString sColorString) {
```

Comment:
I would go with `selectColor()` alone, I think the presence-info is not required in that place, falling back to transparent makes most sense for backwards-compat. And an empty string will be converted to fully transparent by QT.